c++/cpp2py/converters/tuple.hpp

```diff
@@ -11,28 +11,28 @@ namespace cpp2py {
     using tuple_t = std::tuple<Types...>;
 
     // c2py implementation
-    template <std::size_t... Is> static PyObject *c2py_impl(tuple_t const &t, std::index_sequence<Is...>) {
-      auto objs        = std::array<pyref, sizeof...(Is)>{pyref(py_converter<Types>::c2py(std::get<Is>(t)))...};
+    template <std::size_t... Is> static PyObject *c2py_impl(tuple_t t, std::index_sequence<Is...>) {
```

Comment:
you can simply say `auto ... Is`, it will pick up the right integer automatically ...
no need of std:;size_t or size_t or ....